x-pack/test_serverless/api_integration/test_suites/security/cloud_security_posture/telemetry.ts

```diff
@@ -106,7 +94,23 @@ export default function ({ getService }: FtrProviderContext) {
         roleAuthc,
         internalRequestHeader
       );
-      await waitForPluginInitialized(supertestWithoutAuth, internalRequestHeader, roleAuthc);
+
+      log.debug('Check CSP plugin is initialized');
+      await retry.try(async () => {
+        const supertestAdminWithHttpHeaderV1 = await roleScopedSupertest.getSupertestWithRoleScope(
+          'admin',
+          {
+            useCookieHeader: true,
+            withInternalHeaders: true,
+            withCustomHeaders: { [ELASTIC_HTTP_VERSION_HEADER]: '1' },
+          }
+        );
+        const response = await supertestAdminWithHttpHeaderV1
+          .get('/internal/cloud_security_posture/status?check=init')
+          .expect(200);
+        expect(response.body).to.eql({ isPluginInitialized: true });
+        log.debug('CSP plugin is initialized');
+      });
```

Comment:
move it here for better visibility `supertestAdminWithHttpHeaderV1` vs `supertestAdminWithCookieCredentials`